test/kubemark/start-kubemark.sh

```diff
@@ -370,6 +370,7 @@ echo -e "${color_yellow}STARTING SETUP FOR MASTER${color_norm}"
 find-release-tars
 create-master-environment-file
 create-master-instance-with-resources
+MASTER_IP_AND_PORT="${MASTER_IP_AND_PORT:-$MASTER_IP}"
```

Comment:
Since I changed a few variables from ```MASTER_IP``` to ```MASTER_IP_AND_PORT``` in ```start-kubemark.sh```, ```MASTER_IP_AND_PORT``` will need to be set to ```MASTER_IP``` in the case that the dev is not using the pre-existing provider.  That's what this does. But I could also do what I mentioned above.  I could document when using the pre-existing provider for the dev to include the port into the ```MASTER_IP``` variable.